base/rational.jl

```diff
@@ -375,8 +375,8 @@ function ^(x::Rational, n::Integer)
 end
 
 ^(x::Number, y::Rational) = x^(y.num/y.den)
-^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T, y.num / y.den))
-^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T, y.num / y.den))
+^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
+^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
```

Comment:
My thinking here was that there is a higher chance to have a conversion from `Int` then from `Rational`.  But if you think it's ok to have `x^convert(T,y)` I can change it.